### ISSUE TITLE
Release 0.28.3

### ISSUE THREAD

**Issue by Mingun** [tafia/quick-xml#607]
Checklist (minimal list of actions for cutting release):

- [ ] Merge #508?
- [ ] `$env:RUSTDOCFLAGS="--cfg docs_rs"; cargo +nightly doc --all-features` and check generated documentation for missing / unclear things
- [ ] Update version in `Cargo.toml`
- [ ] Update `Changelog.md` with date of release, add new empty Unreleased headings
- [ ] `cargo package` for final check
- [ ] Push `master` to my fork, wait while CI pass. Repeat with force pushs if necessary
- [ ] `cargo publish`
- [ ] Create and push tag `v0.28.3` and push `master` to upstream
- [ ] Create a Release on GitHub (in GitHub UI)

**Comment by dralley** [comment]
#508 is a feature, I would expect it would go into 0.29

**Comment by Mingun** [comment]
Yes, it is. Actually, semver allows as release either `0.28.3` or `0.29.0`. I just thinking, that because we have not breaking changes in current master and in #508, we could do a patch release (the previous `0.28.1` and `0.28.2` I'd made with the same rationale). But if you insist, lets release as `0.29.0`

**Comment by dralley** [comment]
That's not my understanding of semver, my understanding is that patch releases are for bugfixes rather than functionality.  I don't care much though.

**Comment by Mingun** [comment]
Because I just merged #612 which adds new `Error` variant, now the target version will be 0.29.0.

@dralley, I plan to make release tomorrow, approximately 15-20 hours later. So if you have time to finish with #609 before that time -- OK. No - well, okay, will be included in the next release

**Comment by dralley** [comment]
I won't have time, I have some items at work to deal with first.  It's fine to put it in the next release.
